packages/strapi-plugin-upload/admin/src/utils/formatFileForEditing.js

```diff
@@ -7,7 +7,8 @@ const formatFileForEditing = file => {
     abortController,
     id: file.id,
     file: {
-      ...pick(file, ['size', 'ext', 'width', 'height', 'url', 'mime', 'name']),
+      ...pick(file, ['size', 'ext', 'width', 'height', 'mime', 'name']),
+      url: get(file, ['formats', 'thumbnail', 'url']),
```

Comment:
I'm not sure but is it the URL that is used when the user needs to copy the URL of the file? If yes, should we override the URL here?

Reply:
Good catch i'll add another info!

Reply:
In fact here it's best to use the url